includes/utilities.inc

```diff
@@ -136,6 +140,7 @@ function islandora_iiif_manifests_get_object_canvas($object) {
   $canvas_object['thumbnail_uri'] = $image_info['thumbnail_uri'];
   $canvas_object['resource_uri'] = $image_info['resource_uri'];
   $canvas_object['service_uri'] = $image_info['service_uri'];
+  $canvas_object['id_uri'] = $image_info['id_uri'];
   $canvas_object['imageWidth'] = $image_info['imageWidth'];
   $canvas_object['imageHeight'] = $image_info['imageHeight'];
```

Comment:
Sure that all these exist?

Reply:
Yes, though imageWidth and imageHeight depends on if the object has either a TECHMD or OBJ. The width and height are retrieved from either one of those datastreams. I assume that both datastreams are alway available. Or is that not the case?